Sources/swift-build/main.swift

```diff
@@ -99,6 +99,12 @@ do {
         let (rootPackage, externalPackages) = try fetch(opts.path.root)
         dumpDependenciesOf(rootPackage: rootPackage, mode: mode)
 
+    case .ShowPackage:
+        let root = opts.path.root
+        let manifest = try parseManifest(path: root, baseURL: root)
+        let out = manifest.package.toTOML()
+        print(out)
```

Comment:
My vote is that we only ever define this feature in terms of dumping the Package as JSON. I think that is the only format we would want to support externally.